pages/schedule.js

```diff
@@ -14,13 +12,15 @@ export default function SchedulePage({ schedule }) {
   );
 }
 
-export async function getStaticProps() {
-  const schedule = await airtable.getSchedule();
+export async function getServerSideProps({ query }) {
+  // Cache isn't available locally, only in a Cloudflare Workers environment
+  const cache = typeof caches !== "undefined" ? caches?.default : undefined;
```

Comment:
This is annoying. Ideally we use wrangler to run locally which would have this available to us, but there's no easy way to do this with Next.js so we're stuck with Node.

There may be a way to mock the Cache API locally, I'll take a look at this.